app/discord.js

```diff
@@ -0,0 +1,14 @@
+const Discord = require('discord.js');
+const hookid = process.env.DISCORD_WEBHOOK_URl.split('/')[5];
+const hooktoken = process.env.DISCORD_WEBHOOK_URl.split('/')[6];
```

Comment:
These will return an error if users update the bot, but don't add `DISCORD_WEBHOOK_URl=""` to the .env file. I'm not sure if we should worry about that though. @Nicklason ?

Reply:
I thought about adding a shorthand conditional statement here. Decided not to, if they don’t want to use the discord msgs they should just leave it at “”

Reply:
It should not make the bot crash if you don’t include it, so please fix that